### DISCUSSION TITLE
Different volume normalization: files system vs Spotify

### DISCUSSION THREAD

**Discussion by AndreasBVB** [music-assistant/support#927] [Q&A] [answered]
Hi I mix local music files with Spotify songs into one playlist and find this is a cool feature. However I usually find the Spotify songs much louder than local songs, so have to tune the volume up and down as the list shuffles. Did I miss anything?

BTW: my local MP3 files all have track & album replay gains nicely scanned and stored in tags. MA doesn't seem to respect these numbers. I'm actually curious of how MA determines the normalization gains to use.

**Comment by OzGav** [comment]
Do you have normalisation turned on? QUEUE>>THREE DOTS TOP RIGHT>>SETTINGS>>ADVANCED SETTINGS

edit: there is some reference to how this works here https://github.com/music-assistant/hass-music-assistant/issues/287

**Comment by AndreasBVB** [comment, marked as answer]
It's on. Normalization works well both within local or within Spotify. Not normalizing songs from the two platforms to the close enough level though.

**Reply by OzGav** [reply]
Marcel has advised me this “ A song has to be played one time before its loudness is known/calculated. Until then an educated guess is done and I guess in this case its based on the wrong average. It will get better as more music is played as it will get the average of more songs from the same provider to use as default”

So try and play the same songs again between the providers and see if it improves.

**Reply by AndreasBVB** [reply]
Thanks for the info! This makes sense.
